cmd/instance/instance_list.go

```diff
@@ -61,7 +61,9 @@ If you wish to use a custom format, the available fields are:
 		ow := utility.NewOutputWriter()
 		for _, instance := range instances {
 			ow.StartLine()
-
+			if len(instances) == 0 {
```

Comment:
I think this should move before we run the for loop. Otherwise, this would never get executed

Reply:
Fixed this 👍🏼 
Put up the for loop in the else statement for the if